### ISSUE TITLE
no fallback to 'GENERIC' icon list of vendor data doesn't provide one

### ISSUE THREAD

**Issue by caolanm** [rpminspect/rpminspect#583]
So the attached example fails with "1) Desktop file /usr/share/applications/libreoffice-startcenter.desktop on x86_64 references icon document-new but no subpackages contain document-new" on fedora/centos/rhel configurations but is ok with --release='GENERIC'
[desktopfilestest-1-1.fc35.src.rpm.zip](https://github.com/rpminspect/rpminspect/files/7710412/desktopfilestest-1-1.fc35.src.rpm.zip)

**Comment by dcantrell** [comment]
This was happening because I neglected to install the /usr/share/rpminspect/icons/* files in the rpminspect data package.  I noted in the internal bug report that I fixed that in rpminspect-data-redhat and a new build kicked off.

I modified the desktop inspection to get a list of standard icon names from /usr/share/rpminspect/icons/PRODUCT where PRODUCT matches the product release string rpminspect either derives on its own or you specify on the command line.  For example, Fedora 35 would be "fc35".

I noticed the same problem I made in the rpminspect-data-fedora and rpminspect-data-centos packages, so I fixed those too.  New Copr builds are available and the next release I do of the data packages will contain the fix as a stable build.
